zhaquirks/__init__.py

```diff
@@ -66,18 +66,37 @@ class LocalDataCluster(CustomCluster):
 
     async def bind(self):
         """Prevent bind."""
+        self.log(
+            logging.DEBUG,
+            f"binding LocalDataCluster with cluster_id: {self.cluster_id}",
+        )
         return (foundation.Status.SUCCESS,)
 
     async def unbind(self):
         """Prevent unbind."""
+        self.log(
+            logging.DEBUG,
+            f"unbinding LocalDataCluster with cluster_id: {self.cluster_id}",
+        )
         return (foundation.Status.SUCCESS,)
 
     async def _configure_reporting(self, *args, **kwargs):  # pylint: disable=W0221
         """Prevent remote configure reporting."""
+        self.log(
+            logging.DEBUG,
+            f"configuring reporting for LocalDataCluster with cluster_id: {self.cluster_id}",
+            *args,
+            **kwargs,
+        )
         return (foundation.ConfigureReportingResponse.deserialize(b"\x00")[0],)
 
     async def read_attributes_raw(self, attributes, manufacturer=None):
         """Prevent remote reads."""
+        msg = f"reading attributes for LocalDataCluster with cluster_id: {self.cluster_id}"
```

Comment:
It's probably fine, but this might be logged a lot (in deubg). Like for Xiaomi plugs with multiple `LocalDataCluster`s when they're polled.

Reply:
At debug we log all the traffic anyway so this is an outlier…